cos-fleet-catalog-tools/camel-connector-maven-plugin/src/generated/resources/schema/camel-yaml-dsl-restricted.json

```diff
@@ -0,0 +1,308 @@
+{
+  "$schema" : "http://json-schema.org/draft-04/schema#",
```

Comment:
is this file auto generated in this repo ?

Reply:
Yes, this is the output from this plugin execution
https://github.com/bf2fc6cc711aee1a0c2a/cos-fleet-catalog-camel/pull/530/files/666c74b48bc798bd54cd40f8ccd5cc72a20c45ef#diff-1658422f830fa328b8f1e0e616da0902c8810a180a62479e1ad9f18f99ce51f4R343

Reply:
oh, I missed it